.github/workflows/publish_alpha.yml

```diff
@@ -0,0 +1,71 @@
+# This workflow will generate a distribution and upload it to PyPI
+
+name: Publish Alpha Build ...aX
+on:
+  push:
+    branches:
+      - dev
+    paths-ignore:
+      - 'version.py'
+      - '.github/**'
+      - '.gitignore'
+      - 'LICENSE'
+      - 'CHANGELOG.md'
+      - 'MANIFEST.in'
+      - 'README.md'
+      - 'scripts/**'
+  workflow_dispatch:
+
+jobs:
+  build_and_publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: dev
+          fetch-depth: 0 # otherwise, there would be errors pushing refs to the destination repository.
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+      - name: Install Build Tools
+        run: |
+          python -m pip install build wheel
+      - name: Increment Version
+        run: |
+          VER=$(python setup.py --version)
+          python scripts/bump_alpha.py
```

Comment:
you need to either move to the Neon shared actions, or add the scripts folder also